byterun/pyvm2.py

```diff
@@ -579,9 +621,28 @@ def byte_BUILD_SET(self, count):
         elts = self.popn(count)
         self.push(set(elts))
 
-    def byte_BUILD_MAP(self, size):
-        # size is ignored.
-        self.push({})
+    def byte_BUILD_CONST_KEY_MAP(self, count):
+        # count values are consumed from the stack.
+        # The top element contains tuple of keys
+        # added in version 3.6
+        keys = self.pop()
+        values = self.popn(count)
+        kvs = dict(zip(keys, values))
+        self.push(kvs)
+
+    def byte_BUILD_MAP(self, count):
+        # Pushes a new dictionary on to stack.
+        if not(six.PY3 and sys.version_info.minor >= 5):
```

Comment:
this might be more clear as `sys.version_info[:2] < (3, 5)`. at first I thought this was flipped

Reply:
I will update that